react/index.d.ts

```diff
@@ -2140,7 +2140,187 @@ declare namespace React {
+        numOctaves?: number | string;
+        operator?: number | string;
+        order?: number | string;
+        orient?: number | string;
+        orientation?: number | string;
+        origin?: number | string;
+        overflow?: number | string;
+        overlinePosition?: number | string;
+        overlineThickness?: number | string;
+        paintOrder?: number | string;
+        panose1?: number | string;
+        pathLength?: number | string;
+        patternTransform?: number | string;
+        pointerEvents?: number | string;
+        pointsAtX?: number | string;
+        pointsAtY?: number | string;
+        pointsAtZ?: number | string;
+        preserveAlpha?: number | string;
+        primitiveUnits?: number | string;
+        radius?: number | string;
+        refX?: number | string;
+        refY?: number | string;
+        renderingIntent?: number | string;
+        repeatCount?: number | string;
+        repeatDur?: number | string;
+        requiredExtensions?: number | string;
+        requiredFeatures?: number | string;
+        restart?: number | string;
+        rotate?: number | string;
+        scale?: number | string;
+        seed?: number | string;
+        shapeRendering?: number | string;
+        slope?: number | string;
+        spacing?: number | string;
+        specularConstant?: number | string;
+        specularExponent?: number | string;
+        speed?: number | string;
+        startOffset?: number | string;
+        stemh?: number | string;
+        stemv?: number | string;
+        stitchTiles?: number | string;
+        strikethroughPosition?: number | string;
+        strikethroughThickness?: number | string;
+        string?: number | string;
+        strokeDashoffset?: number | string;
+        surfaceScale?: number | string;
+        systemLanguage?: number | string;
+        tableValues?: number | string;
+        targetX?: number | string;
+        targetY?: number | string;
+        textDecoration?: number | string;
+        textRendering?: number | string;
+        textLength?: number | string;
+        to?: number | string;
+        u1?: number | string;
+        u2?: number | string;
+        underlinePosition?: number | string;
+        underlineThickness?: number | string;
+        unicode?: number | string;
+        unicodeBidi?: number | string;
+        unicodeRange?: number | string;
+        unitsPerEm?: number | string;
+        vAlphabetic?: number | string;
+        vHanging?: number | string;
+        vIdeographic?: number | string;
+        vMathematical?: number | string;
+        vectorEffect?: number | string;
+        vertAdvY?: number | string;
+        vertOriginX?: number | string;
+        vertOriginY?: number | string;
+        viewTarget?: number | string;
+        visibility?: number | string;
+        widths?: number | string;
+        wordSpacing?: number | string;
+        writingMode?: number | string;
+        xHeight?: number | string;
+
+        // old attrs
```

Comment:
What's the difference between "old" and "new"? Why the separation? I'd like the lists to be merged; otherwise, future authors won't know where to add attributes.